shamuna/van.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="van.css">
+</head>
+<body>
+ <div class="header">
+        
+            <a href="#">HOME</a>
+           <a href="#">Tickets&visit</a>
+            <a href="#"> about</a>
+            <a href="#">Webstore</a>
+           <a href="#">NL/EN</a>
+         
+    </div>
+   
+    <div class="container">
+      <div class="small-container">
+        <div class="one">
+            <h2>Art</h2>
+<p>Explore Vincent van Gogh's famous paintings and drawings, work by his artist friends and contemporaries, French and Japanese prints and more.</p>
+        
+        </div>
+        <div class="two">
+      <img src="C:\Users\46767\Desktop\shamuss\frontend-dragons\van\img\600x789.webp">
+        </div>
+        <div class="three">
+        <img src="C:\Users\46767\Desktop\shamuss\frontend-dragons\van\img\600x658.webp">
+        </div>
+        <div class="four">
+         <img src="C:\Users\46767\Desktop\shamuss\frontend-dragons\van\img\ac_van_gogh_slider_comp11.jpg">
+        </div>
+        <div class="five">
+          <img src="C:\Users\46767\Desktop\shamuss\frontend-dragons\van\img\800x1028.webp">
```

Comment:
This path won't work for anyone else, use a relative path '../img/something.jpg'

Reply:
k